paddle/fluid/operators/activation_mkldnn_op.cc

```diff
@@ -23,6 +24,18 @@ using paddle::framework::Tensor;
 using paddle::platform::MKLDNNDeviceContext;
 
 namespace {
+std::string gethash(const mkldnn::memory::dims &operand_dims,
+                    const mkldnn::algorithm algorithm) {
+  auto dim2str = [](const mkldnn::memory::dims &operand_dims) {
```

Comment:
Is that possible two fc ops with same src dims, then would this be right?  Then would share same hash code?

Reply:
Probably yes, but they will be reusable. I improved hash code for input data (key_src_data)

Reply:
OK, good.